### ISSUE TITLE
LinterRubocop.Linter is deprecated.

### ISSUE THREAD

**Issue by arturhoo** [AtomLinter/linter-flake8#53]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
LinterFlake8.Linter (/Users/artur/.atom/packages/linter/lib/linter.coffee:60:4)
new LinterFlake8 (/Users/artur/.atom/packages/linter-flake8/lib/linter-flake8.coffee:25:4)
Object.lint (/Users/artur/.atom/packages/linter/lib/legacy.coffee:61:21)
<unknown> (/Users/artur/.atom/packages/linter/lib/editor-linter.coffee:69:25)
currentLinter (/Users/artur/.atom/packages/linter/lib/editor-linter.coffee:68:19)
<unknown> (/Users/artur/.atom/packages/linter/lib/editor-linter.coffee:82:22)
```

**Comment by arturhoo** [comment]
I don't know why it was reported with the LinterRubocop scope, but this is from the flake8 plugin.

**Comment by TWHarr** [comment]
getting this is as well

**Comment by webdrone** [comment]
same

**Comment by Arcanemagus** [comment]
Duplicate of #51.
